### ISSUE TITLE
Error in 2.30.0 - ERROR in Template execution failed: TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined

### ISSUE THREAD

**Issue by billneff79** [jantimon/html-webpack-plugin#753]
After upgrading from 2.29.0 to 2.30.0 this morning, we are seeing this error in our build.  This does not occur in 2.29.0.  Was there supposed to be a breaking change in 2.30.0?

```
ERROR in Template execution failed: TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined

ERROR in   TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined

  - template.ejs:89 ../packages/@zimbra/webpack-config/node_modules/ejs-loader/index.js!../packages/@zimbra/webpack-config/dist/template.ejs    .module.exports
    /Users/bneff/repos/zimbra-app/packages/@zimbra/webpack-config/dist/template.ejs:89:25

  - index.js:264
    [webpack-config]/[html-webpack-plugin]/index.js:264:16

  - util.js:16 tryCatcher
    [webpack-config]/[bluebird]/js/release/util.js:16:23

  - promise.js:512 Promise._settlePromiseFromHandler
    [webpack-config]/[bluebird]/js/release/promise.js:512:31

  - promise.js:569 Promise._settlePromise
    [webpack-config]/[bluebird]/js/release/promise.js:569:18

  - promise.js:606 Promise._settlePromiseCtx
    [webpack-config]/[bluebird]/js/release/promise.js:606:10

  - async.js:138 Async._drainQueue
    [webpack-config]/[bluebird]/js/release/async.js:138:12

  - async.js:143 Async._drainQueues
    [webpack-config]/[bluebird]/js/release/async.js:143:10

  - async.js:17 Immediate.Async.drainQueues
    [webpack-config]/[bluebird]/js/release/async.js:17:14
```

**Comment by mastilver** [comment]
Hi @billneff79 

I don't believe there is a breaking change in the new version, here's the diff if you can spot anything: https://github.com/jantimon/html-webpack-plugin/compare/2cc10df68f801f9c329ef4546693c6e0188aa1de...09a96f5500e8294bfa40efa34e12aa178349543d


Would you mind posting your webpack config? Also can you double check it's working on 2.29.0 
and it's not another dependency causing the issue?

**Comment by billneff79** [comment]
I verified that it is purely the switch from 2.29.0 to 2.30.0 that is causing it by doing the following:

```
npm i html-webpack-plugin@2.29.0 && npm run build # all is well
npm i html-webpack-plugin@2.30.0 && npm run build # breakage
```

We have a pretty complicated webpack config.  I'll see if I can get it up here.

**Comment by mastilver** [comment]
@billneff79 a git repository would be even more awesome, that way I can check which commit introduced the issue :)

**Comment by exp10r3r** [comment]
Same issue with us as well. Started happening today. We have a starter app created via preact-cli.

**Comment by mastilver** [comment]
Thank you @exp10r3r, I managed to reproduce the issue

**Comment by developit** [comment]
I may be the common denominator there, let me know if either of the templates are doing something incorrect: https://github.com/developit/preact-cli/blob/master/src/resources/template.html

**Comment by mastilver** [comment]
So this was introduced by #723

**Comment by mastilver** [comment]
#754 fixes the issue, I'm just waiting for @jantimon to release it and we would be good :)

**Comment by mastilver** [comment]
@developit It **might** be caused by this line: https://github.com/developit/preact-cli/blob/master/src/resources/template.html#L13, but I don't believe there is anything wrong with it

I will add some tests later to prevent this kind of changes to land in the future

**Comment by billneff79** [comment]
@mastilver thank you for the quick response!  Much appreciated.

**Comment by jantimon** [comment]
Thanks for your quick feedback I merged #754 and released 2.30.1
